skorch/callbacks/training.py

```diff
@@ -90,3 +91,88 @@ def on_epoch_end(self, net, **kwargs):
             if net.verbose > 0:
                 print("Checkpoint! Saving model to {}.".format(target))
             net.save_params(target)
+
+
+class EarlyStopping(Callback):
+    """Stop training early if a specified `monitor` metric did not improve in
+    `patience` number of epochs by at least `threshold`
+
+    Parameters
+    ----------
+    monitor : str (default='valid_loss')
+      Value of the history to monitor to decide whether to stop training or not.
+      The value is expected to be double and is commonly provided by scoring
+      callbacks such as :class:`skorch.callbacks.EpochScoring`.
+
+    lower_is_better : bool (default=True)
+      Whether lower scores should be considered better or worse.
+
+    patience : int (default=5)
+      Number of epochs to wait for improvement of the monitor value until
+      the training process is stopped.
+
+    threshold : int (default=1e-4)
+      Ignore score improvements smaller than `threshold`.
+
+    threshold_mode : str (default='rel')
+        One of `rel`, `abs`. In `rel` mode,
+        dynamic_threshold = best * ( 1 + threshold ) in 'max'
```

Comment:
Usage of 'min' and 'max' mode terminology here is confusing, because it sounds like these are two more possible options for `threshold_mode`.

Reply:
Fixed (i hope)